databricks_cli/dbfs/cli.py

```diff
@@ -53,7 +55,10 @@ def ls_cli(api_client, l, absolute, dbfs_path): #  NOQA
         dbfs_path = dbfs_path[0]
     else:
         error_and_quit('ls can take a maximum of one path.')
-    files = DbfsApi(api_client).list_files(dbfs_path)
+
+    DbfsApi(api_client).list_files(dbfs_path, is_recursive=recursive)
+    absolute = absolute or recursive
+
     table = tabulate([f.to_row(is_long_form=l, is_absolute=absolute) for f in files],
```

Comment:
I think you have meant to define a files variable here since we are trying to loop over it? Is there a definition I have missed?

Reply:
fixed with b0fb9cfa9360a87eae5d2c91e6cc076a9a184205